faf-commons-data/src/main/java/com/faforever/commons/replay/shared/LuaData.java

```diff
@@ -3,9 +3,43 @@
 import java.util.Map;
 
 public sealed interface LuaData {
-  record Number(float value) implements LuaData {}
-  record String(java.lang.String value) implements LuaData {}
   record Nil() implements LuaData {}
-  record Table(Map<java.lang.String, LuaData> value) implements LuaData {}
-  record Bool(boolean value) implements LuaData {}
+  record Number(float number) implements LuaData {}
+  record String(java.lang.String string) implements LuaData {}
+  record Bool(boolean bool) implements LuaData {}
```

Comment:
I would just keep the variables named value. It seems more clear and no need to repeat the type a third time.

Reply:
Processed in https://github.com/FAForever/faf-java-commons/pull/148/commits/ff248843430247be2267efdbcdf4c7ea1aba6842